app/services/payment_provider_customers/gocardless_service.rb

```diff
@@ -38,24 +50,23 @@ def access_token
       organization.gocardless_payment_provider.access_token
     end
 
+    def client
+      @client || GoCardlessPro::Client.new(access_token: access_token, environment: :sandbox)
```

Comment:
do we want to commit the sandbox env? what the strategy to put it in production?

Reply:
is `@client` initialized before?

Reply:
I suggest to have the `environment: :sandbox` defined at payment provider level (it could be a simple method for now). That way if we want to turn it as a config option of the payment provider instance it's easy :)

```ruby
module PaymentProviders
  class GocardlessProvider < BaseProvider
    # ...

    def environment
      :sandbox # Should be turned as a config option of the model
    end

    # ...
  end
end
```

Reply:
For production you can pass either `:live` param or don't pass it at all. My initial plan was to remove it for production later. But make sense to have it on one place so that we don't forget to remove it :) Will add it on provider level

Reply:
updated:)